packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -60,27 +58,11 @@ const contract = async (
 
   const creatorFacet = prepareChainHubAdmin(zone, chainHub);
 
-  // TODO should be a provided helper
-  /** @type {(brand: Brand) => Vow<VBankAssetDetail>} */
-  const findBrandInVBank = vowTools.retriable(
```

Comment:
Do I correctly understand we are blocked on implementing this due to brand/object identity mismatch inside and outside the membrane?

Reply:
The motivations to switch from "look up this brand" to "give me all of them" are (1) "look up this brand" fetches all of them anyway, and (2) there are other things the caller might want to do besides look up just one brand. For example:

```js
    const vbank = await orch
      .getChain('agoric')
      .then(ch => ch.getVBankAssetInfo())
      .then(as => Object.fromEntries(as.map(a => [a.issuerName, a])));
    t.log('vbank assets', Object.keys(vbank));

    const give = { USDC: AmountMath.make(vbank.USDC.brand, 100n) };
    const offerArgs = { staked: AmountMath.make(vbank.ATOM.brand, 200n) };
```

Reply:
Cool - makes sense! Ty